locale/override/en.po

```diff
@@ -1,3 +1,3 @@
-msgctxt "field:gnuhealth.patient,general_info:"
-msgid "General Information"
-msgstr "General Observations"
+msgctxt "field:gnuhealth.patient,dob:health."
+msgid "DoB"
```

Comment:
The `dob:health` part of the msgctxt causes the link to the XML templates defined below? The msgid doesn't have to match the `name` in the template? I mean `dob` vs. `DoB`

Reply:
This one was messy, basically we extend an existing module, and this field is in the other module hence why this translation goes into /override/ directory.
It changes the translation for a field that's not in this repository, http://hg.savannah.gnu.org/hgweb/health/file/ebdbbea9e7e0/tryton/health/health.py#l389
That field is defined as `dob = fields.Date('DoB', help='Date of Birth')`
So the `dob` in the `msgctxt` refers to the field, and `DoB` refers to the source string that it was given in gnuhealth.

:scream_cat:

Reply:
I'm having a similar issue with the `active` field, except that one is done differently again!

Reply:
forgot to mention, the `health` part is because the module we're overriding is called `health`. This is an extra part to `msgctxt` that we should only need when overriding